content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,201 +2,234 @@
+
 ---
 
 ## Overview
-Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution, now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
+
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog
+provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution,
+now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom
+metrics, and direct log collection.
 
   <div class="alert alert-warning">This feature is in public beta. You can provide feedback through a <a href="https://forms.gle/HSiDGnTPvDvbzDAQA">feedback form</a>, or through your standard support channels. During the beta period, Cloud Run monitoring and APM tracing are available without a direct cost. Existing APM customers may incur increased span ingestion and volume costs. </div>
 
-## Tracing and custom metrics
-### Build your container
+## Getting started
+
+### Prerequisites
+
+Make sure you have a [Datadog API Key][10] and are using a programming language [supported by a Datadog tracing library][2].
+
+### Deploy a sample application in one click
+
+To deploy a sample application without the need to follow the rest of the guide, you can use on of [these examples][12]. The installation process will ask the required details such as the Datadog API Key. Note that these applications send data to the default website `datadoghq.com`.
 
-If you are using a Dockerfile to build your application, complete the following:
+### Configure your container with the Datadog Agent
 
-1. Instrument your application with a [supported Datadog tracing library][2]
+To build your container with Datadog instrumentation you can follow one of these two methods, depending if you are using a Dockerfile or buildpack.
 
-2. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
+#### Configure the Datadog Agent using Dockerfile
 
-3. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
+Using a Dockerfile to build your container, the following steps are needed:
 
-4. Use the `CMD` instruction to run your existing application and other required commands as arguments.
+1. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
 
-The following are examples of how to complete these three steps. You may need to adjust these examples depending on your existing Dockerfile setup. 
+2. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
```

Comment:
Related to my other comment about pulling in key changes from the sample repo to the sample snippet below, I think it might be better to merge these explanation as comments to the dockerfile sample?

Something in my mind (not tested):

```
// Install and run the Datadog Agent
COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
ENTRYPOINT ["/app/datadog-init"]

// Install the Datadog tracing library
pip install ddtrace

// Start your application with Datadog tracing
CMD ["ddtrace-run", "python", "app.py"] (adapt this line to your needs)

// Configure Datadog tags
ENV DD_SERVICE=my-service
ENV DD_ENV=dev
ENV DD_VERSION=1
```

Reply:
yeah it feel like a duplicate instruction in the list and code. I changed it in the last comment